hackathon_site/dashboard/frontend/src/components/general/Navbar/Navbar.js

```diff
@@ -2,31 +2,30 @@ import React from "react";
 import styles from "./Navbar.module.scss";
 import { Link } from "react-router-dom";
 // Images and logos
-import Home from "@material-ui/icons/Home";
-import ShoppingCart from "@material-ui/icons/ShoppingCart";
+import DashboardIcon from "@material-ui/icons/Dashboard";
+import LocalMallIcon from "@material-ui/icons/LocalMall";
 import Notifications from "@material-ui/icons/Notifications";
 import ListAlt from "@material-ui/icons/ListAlt";
-import AccountCircle from "@material-ui/icons/AccountCircle";
+import AccountBoxIcon from "@material-ui/icons/AccountBox";
 import InsertChartOutlined from "@material-ui/icons/InsertChartOutlined";
 import { ReactComponent as Inventory } from "assets/images/icons/Hardware.svg";
 // Components
 import Button from "@material-ui/core/Button";
 import IconButton from "@material-ui/core/IconButton";
 import Hidden from "@material-ui/core/Hidden";
 
-const Navbar = () => (
+export const Navbar = ({ cartQuantity }) => (
     <nav className={styles.nav}>
         <Hidden implementation="css" smDown>
```

Comment:
Likewise to my comment on the header component, can we leverage the `Hidden` there instead of needing it here too?